lib/ratelimit.c

```diff
@@ -104,3 +104,14 @@ log_ratelimit_allow(struct log_ratelimit_state *lrs)
 
 	return false;
 }
+
+void
+update_log_ratelimit_level(uint32_t level, unsigned int lcore_id)
+{
+	struct log_ratelimit_state *lrs;
+
+	RTE_VERIFY(lcore_id < RTE_MAX_LCORE);
+
+	lrs = &log_ratelimit_states[lcore_id];
+	lrs->ratelimit_level = level;
```

Comment:
Shouldn't we update the loglevel information in DPDK as well here to avoid a mismatch?